lte/gateway/configs/control_proxy.yml

```diff
@@ -50,3 +50,17 @@ sentry_upload_mme_log: false
 # Rate at which we want to sample Python error events
 # Should be a number between 0 (0% of errors sent) and 1 (100% of errors sent)
 sentry_sample_rate: 1.0
+# Message patterns that are excluded from Sentry (regex substring match)
+sentry_excluded_errors:
```

Comment:
@christinewang5 recently added the sentry config to the mconfig, which can be streamed down from the Orc8r. 
Let's add any new sentry configs there instead. https://github.com/magma/magma/blob/master/orc8r/protos/mconfig/mconfigs.proto#L25

if you have questions on the how the config gets streamed down from the orc8r, we can meet and talk through it. :) 
But the quick gist is:
1. The API model is defined here: https://github.com/magma/magma/blob/master/orc8r/cloud/go/services/orchestrator/obsidian/models/swagger.v1.yml#L1435
2. This gets generated into a go struct here: https://github.com/magma/magma/blob/master//orc8r/cloud/go/services/orchestrator/obsidian/models/network_sentry_config_swaggergen.go#L18
3. The values are passed onto the mconfig here: https://github.com/magma/magma/blob/master//orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go#L104
4. This struct then gets streamed down and is processed at https://github.com/magma/magma/tree/master/orc8r/gateway/python/magma/configuration
5. The file ends up in `/var/opt/magma/configs/gateway.mconfig` the default values are taken from `/etc/magma/gateway.mconfig`

^ This shared config to store config used by all services is a new concept, so you might have to add functions in the files in step4 to fetch this value.

Reply:
I think I've done the first three steps but I'm unsure about 4 and 5. From what I can see, the Python Sentry configuration is still completely based on control_proxy.yml. Should this be changed to use mconfig here?

Reply:
Another thing I've noticed is that the two URL fields are still called "URL" in the swagger interface but were renamed to "DSN" in the proto interface. Is that intentional?

Reply:
Yes, we have to make the change to read from the mconfig. (So this will have to be a separate PR. We did something similar for C++ way back. We can take values from the Mconfig UNLESS it is overwritten by the control_proxy files.

Reply:
Is #10170 the issue to do that?

Reply:
https://github.com/magma/magma/issues/9508 is closer. We don't seem to have filed a task for modifying the python config part.

Reply:
At the moment, we've added the new exclusion list to both mconfig and the control_proxy.yml. Since all the Sentry config is not yet used in the services, I would propose to keep it in both for this PR and then to have another PR where the Python services are changed to use the mconfig for Sentry.

Should we work on the shared Sentry config or is this already planned by someone else (e.g. @christinewang5 in #10170)?

Reply:
It would be great if you could add another PR to add the code needed to read from shared mconfig. (Please let me know if you need guidance on this).

If the pattern exclusion feature needs to be done quickly, we can add it into control_proxy.yml. However, I am hesitant towards it as control_proxy.yml isn't supposed to be a place for sentry configurations. (I added it there as a quick solution when I initially built out the sentry configs, but probably should have contained it in a separate file.) So if we could avoid adding any more fields there, that would be good.

Reply:
@themarwhal We created a draft (#10557) to get Sentry config from mconfig. Would be great to get some feedback! :)